main.nf

```diff
@@ -909,16 +909,42 @@ workflow.onComplete {
         log.info ""
         log.info("Logs folder: logs_${workflow.sessionId.toString().substring(0,8)}")
         log.info "Cleaning up any temporary files left behind..."
-        file("$workflow.projectDir/paralog_distributions/ks_tmp*", type: "any")
-            .each {
-                result = it.deleteDir();
-                log.info result ? "$it" : "Can't delete: $it"
-            }
-        file("$workflow.projectDir/ortholog_distributions/ks_tmp*", type: "any")
-            .each {
-                result = it.deleteDir();
-                log.info result ? "$it" : "Can't delete: $it"
-            }
+
+        species_name = file("${configfile}").readLines()[1].split()[2]
```

Comment:
This will read whatever is in the second line of the file and will not work if that line does not contain the focal_species parameter because e.g. someone added some comments in the config file before or changed the order of the parameters.
The code should explicitly look for the focal_species parameter value.

Reply:
Done!